reportportal_client/service.py

```diff
@@ -85,6 +85,25 @@ def _get_messages(data):
     return error_messages
 
 
+def _perform_reliably(request, **kwargs):
+    """Fix for https://github.com/reportportal/client-Python/issues/94
+    Attempt to perform provided HTTP request with kwargs,
+    Ingore KeyError exception first HTTP_REQUEST_RETRY_COUNT times.
+    Return a response or propagate last exception if HTTP_REQUEST_RETRY_COUNT exceeded.
+    """
+
+    res = None
+    for _ in range(HTTP_REQUEST_RETRY_COUNT):
+        try:
+            res = request(**kwargs)
+            break
+        except KeyError:
+            pass
+    else:
+        raise  # noqa: pylint: disable=misplaced-bare-raise
```

Comment:
You should not ignore this error. You may raise an incorrect exception in some cases.

```
    error = None
    for attempt in range(HTTP_REQUEST_RETRY_COUNT):
        try:
            return request(**kwargs)
        except KeyError as err:
            error = err
    raise error
```

Reply:
Good point. Thanks